.gitignore

```diff
@@ -4,3 +4,4 @@ env/
 build/
 dist/
 *.egg-info/
+.idea/
```

Comment:
Generally speaking we'd like to keep IDE-specific files out of the global .gitignore. It should probably be added to your global gitignore file.